src/Tribe/Main.php

```diff
@@ -216,12 +236,17 @@ public static function instance() {
 		 * Initializes plugin variables and sets up WordPress hooks/actions.
 		 */
 		protected function __construct() {
-			$this->pluginPath = trailingslashit( dirname( dirname( dirname( __FILE__ ) ) ) );
-			$this->pluginDir  = trailingslashit( basename( $this->pluginPath ) );
-			$this->pluginUrl  = plugins_url( $this->pluginDir );
+			$this->pluginPath = $this->plugin_path = trailingslashit( dirname( dirname( dirname( __FILE__ ) ) ) );
```

Comment:
Not sure what you mean by "to notify users", but why would you want to have a magic `__get` instead of explicit vars? Explicit is almost always a better choice, IMHO.